README.md

```diff
@@ -310,16 +310,117 @@ will be called with the updated state object.
 The returned `removeListener` function de-registers the previously registered
 listener function.
 
-## `getState() #=> state`
+### `getState() #=> state`
 
 Used by components to get the initial state of the store. Returns the currently
 encapsulated state object.
 
-## `setState(state)`
+### `setState(state)`
 
 Explicitly stores the passed in state on the store instance. In the next lesson
 we're going to explore how to generalize data-flow in our application to allow
-actions to mutate stores via a kind of central event bus ("Dispatcher").
+actions to mutate stores via a kind of central event bus ("Dispatcher"). We're
+slowly moving towards an architecture in which stores are being updated using
+external action handlers rather than explicit function calls. But we aren't
+quite there yet!
+
+### Custom Getters
+
+So far our store only has a single method for extracting all the user records
+that have been stored in it.
+
+This can be a bit messy, since in our above example, the `<Profile />` component
+actually stores **all** received user object, even though it only ever renders
+a single one.
+
+In scenarios like that, it's quite common to add this "filtering" logic to the
+underlying store.
+
+Instead of finding the matching user object in the `<Profile />` component's
+render method, we can instead add a new method to the `UserStore` in order to
+enable us to use similar logic in other components that might also render users
+(such as a list of friends):
+
+```js
+class UserStore {
```

Comment:
Can we add a comment indicating that the other methods of `UserStore` are still there? Something like `// ...other methods`, or whatever. Seems silly, but I guess you never know.